### ISSUE TITLE
Game crashes as soon as it detects my steering wheel.

### ISSUE THREAD

**Issue by arisnoGTO** [E66666666/GTAVManualTransmission#122]
I am using a cheap wheel, tracer sierra to play, but from what I checked it should be compatible with the mod, as it also connects to xbox. I have no other mods of any kind right now on my game files and with keyboard the mod works flawlessly.

Update: After a more thorough looking into this matter, I will provide some logs of what happened and will state a few more things I found out. I reinstalled GTA V and the mod and did file verification, so there are no problems there. All files are properly installed. I tried running the WheelSetup.exe and it crashed as well, full log below. My wheel has 2 modes, XINPUT and DINPUT, but they do not seem to make a difference no matter which one is active. The crash happens anyway. I managed to go up to about 20 seconds with my steering wheel recognized by the mod before a crash. All times it didn't immediately do so, it did after I got brave enough to start setting it up (I always start with steering my wheel fully to the right). I can also safely say that this problem is way beyond my capabilities to solve and would appreciate assistance.
[WheelSetup.log](https://github.com/E66666666/GTAVManualTransmission/files/8357713/WheelSetup.log)

Update 2: I was actually not switching wheel modes, it works fine with the other one.

**Comment by E66666666** [comment]
Yeah, seems like this implementation just has trouble with XInput-capable devices.

Sorry you had to go through that, but looks like switching to DInput mode should have solved your issues.

Thanks for the thorough updates!

**Comment by arisnoGTO** [comment]
There is one final update, that being the wheel being seen as a controller by the device and so I was unable to use the binds properly so I decided to uninstall the mod altogether, even though I would be unable to manually switch gears (among various other benefits this mod provided).

Overall I think it is my wheel's flaw here, awesome mod btw I could not believe my eyes when I saw all the features and thanks for getting back to me. Awesome work.
